geometry/Overview.bs

```diff
@@ -215,6 +208,50 @@ and returns a new {{DOMPoint}} object.
 </div>
 
 
+<h3 id=transforming-a-point-with-a-matrix>Transforming a point with a matrix</h3>
+
+<p>To <dfn export>transform a <a>point</a> with a <a>matrix</a></dfn>, given <var>point</var> and
+<var>matrix</var>:
+
+<ol>
+ <li><p>Let <var>x</var> be <var>point</var>'s <a for=point>x coordinate</a>.
+
+ <li><p>Let <var>y</var> be <var>point</var>'s <a for=point>y coordinate</a>.
+
+ <li><p>Let <var>z</var> be <var>point</var>'s <a for=point>z coordinate</a>.
+
+ <li><p>Let <var>w</var> be <var>point</var>'s <a for=point>w perspective</a>.
+
+ <li>
+  <p>Let <var>pointVector</var> be a new column vector with the elements being <var>x</var>,
+  <var>y</var>, <var>z</var>, and <var>w</var>, respectively.
+
+  <p><math display=inline><mrow><mfenced open=[ close=]><mrow><mtable><mtr><mtd><mi>x</mi></mtd></mtr><mtr><mtd><mi>y</mi></mtd></mtr><mtr><mtd><mi>z</mi></mtd></mtr><mtr><mtd><mi>w</mi></mtd></mtr></mtable></mrow></mfenced></mrow></math>
+
+ <li><p><a for=matrix>Post-multiply</a> <var>pointVector</var> with <var>matrix</var>.
```

Comment:
Should this say something about where/how the result is written. I.e this is "matrix * pointVector", but should it be specified as "pointVector = matrix * pointVector" or "pointVector' = matrix * pointVector" or something similar for improved clarity? (The spec isn't entirely consistent in these cases it seems - in some cases the result is assigned and in some cases it isn't.)